testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -388,6 +388,13 @@
   raise "xpath: #{xpath_query} not found" unless find(:xpath, xpath_query, wait: DEFAULT_TIMEOUT)
 end
 
+When(/^I wait until button "([^"]*)" becomes enabled$/) do |text|
+  button = find_button(text, disabled: :all)
+  repeat_until_timeout(message: "Couldn't find button '#{text}'") do
+    break unless button.disabled?
+  end
```

Comment:
```suggestion
  raise "Button '#{text}' still disabled after #{DEFAULT_TIMEOUT} seconds " unless find(text, wait: DEFAULT_TIMEOUT)
```

Reply:
Instead of use ruby code, I prefer we use what Capybara can offers to us :)

Reply:
I see `raise` used in quite a lot of step definitions.

Reply:
You might me got wrong. What I mean is, instead of us that method which loops on a action by waiting using a sleep, we can use the internal timeout of the 'find' Capybara method, as suggested change.